constants.js

```diff
@@ -53,6 +53,7 @@ const respondToReviewCheck = 'respond-to-review-check';
 const oldBuildLabelCheck = 'old-build-label-check';
 
 const checksWhitelist = {
+  // eslint-disable-next-line quote-props
```

Comment:
I have Disabled Lint beacuse while commiting `eslint *.js ./spec/*.js ./lib/*.js actions/**/*.js --fix` runs causing all auto fixable problems to be fixed.